src/Sylius/Bundle/InstallerBundle/Command/SetupCommand.php

```diff
@@ -157,10 +157,17 @@ protected function setupLocale(InputInterface $input, OutputInterface $output)
      */
     protected function setupCurrency(InputInterface $input, OutputInterface $output)
     {
+        $currencyRepository = $this->get('sylius.repository.currency');
         $currencyManager = $this->get('sylius.manager.currency');
         $currencyFactory = $this->get('sylius.factory.currency');
 
         $code = trim($this->getContainer()->getParameter('currency'));
+        $this->currency = $currencyRepository->findOneByCode($code);
```

Comment:
`$this->currency` is used later to setup the channel in `setupChannel()`. Is that what you mean?